<?xml version='1.0'?>
<ns2:test-suite xmlns:ns2='urn:model.allure.qatools.yandex.ru' start='1559805921199' stop='1559805922714'>
    <name>Succussful login Hotline</name>
    <title>Succussful login Hotline</title>
    <test-cases>
        <test-case start='1559805921200' status='failed' stop='1559805922713'>
            <name>loging in</name>
            <title>loging in</title>
            <labels/>
            <parameters/>
            <steps>
                <step start='1559805921201' status='undefined'>
                    <name>step 1</name>
                    <title>step 1</title>
                    <attachments/>
                    <steps>
                        <step start='1559805921202' status='undefined'>
                            <name>step 2</name>
                            <title>step 2</title>
                            <attachments/>
                            <steps/>
                        </step>
                    </steps>
                </step>
            </steps>
            <attachments/>
            <failure>
                <message>Expected 'Hotline - сравнить цены в интернет-магазинах Украины' to be 'Hotline - порівняти ціни в інтернет-магазинах України'.</message>
                <stack-trace>Error: Failed expectation
                    at UserContext.&lt;anonymous> (C:\Users\andriy.oprofat\Desktop\js_test-master\js_test-master\Specs\LoginSuccess.js:19:42)
                    at process._tickCallback (internal/process/next_tick.js:68:7)</stack-trace>
            </failure>
        </test-case>
    </test-cases>
</ns2:test-suite>